src/site/html.rs

```diff
@@ -43,31 +46,32 @@ pub fn build(config: &Config, content: String) -> Result<String> {
             {homepage}
             {favicon}
             {meta_tags}
-            {fringe_css}
+            // {fringe_css}
             {additional_css}
+            <link rel="stylesheet" href="http://localhost:1234/axo-oranda.css"></link>
```

Comment:
Clippy is because of this, will be removed before merge but rn it's better for testing

Reply:
we should come up with a way long term that this is a better dev exp